week08/380. Insert Delete GetRandom O(1).cpp

```diff
@@ -0,0 +1,44 @@
+//TC: O(query)
```

Comment:
What is `query`? It could be `O(1)` per query. Man, you are consistently making great mistakes.

Reply:
Here, query means total_query.